From 8fb9740710294e44b535fb22a053d996e9ed6f81 Mon Sep 17 00:00:00 2001
From: Fugang Duan <B38611@freescale.com>
Date: Fri, 20 Apr 2012 18:27:50 +0800
Subject: ENGR00179636-03 - i.MX6 : Enable ethernet NAPI method in default.

- Enable ethernet NAPI method in default, which can reduce cpu
  loading and RX FIFO overruns in busy system.

Signed-off-by: Fugang Duan <B38611@freescale.com>
---
 arch/arm/configs/imx6_defconfig |    1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/configs/imx6_defconfig b/arch/arm/configs/imx6_defconfig
index 85bdfca..b0a4445 100644
--- a/arch/arm/configs/imx6_defconfig
+++ b/arch/arm/configs/imx6_defconfig
@@ -957,6 +957,7 @@ CONFIG_SMSC911X=y
 # CONFIG_KS8851 is not set
 # CONFIG_KS8851_MLL is not set
 CONFIG_FEC=y
+CONFIG_FEC_NAPI=y
 # CONFIG_FEC_1588 is not set
 # CONFIG_FTMAC100 is not set
 # CONFIG_NETDEV_1000 is not set
-- 
1.7.9.5

